### PR TITLE
Codeception Tests for Users

### PR DESCRIPTION
**Pull request by rmittl**
Pull Request for Issue # .

### Summary of Changes
This PR adds the Codeception Gherkin feature files for Users. We are in process of merging more tests from the GSoC16 repo 
https://github.com/joomla-projects/gsoc16_browser-automated-tests 

### Testing Instructions
These tests have not been included in the builds, but will be included in the next couple of weeks, to run the tests on local system just follow the instruction given on the gsoc16 Repo mentioned above. Users test have been executed on Chrome and they run successfully.

### CODE REVIEW

**Comment by rmittl** [comment]
Adding new User Frontend Scenario as well.

**Review by mbabker — APPROVED** [review]
Looks like a good start, thanks!

**Review by wilsonge — APPROVED** [review]
I'm not a system testing guru but seems good to me too :)

File: tests/codeception/_support/AcceptanceTester.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * @package     Joomla.Test
+ * @subpackage  AcceptanceTester
+ *
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
```
Comment (brianteeman) [inline tests/codeception/_support/AcceptanceTester.php]: 2017

File: tests/codeception/_support/FunctionalTester.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * @package     Joomla.Test
+ * @subpackage  FunctionalTester
+ *
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
```
Comment (brianteeman) [inline tests/codeception/_support/FunctionalTester.php]: 2017

**Comment by rdeutz** [comment]
we merged the test at the infrastructure code sprint with another commit

**Pull request closed without merging**